services/rook-ceph/1.10.1/defaults/cm.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rook-ceph-1.10.1-d2iq-defaults
+  namespace: ${releaseNamespace}
+data:
+  values.yaml: |
+    ---
+    crds:
+      # CRDs are not installed with "crds" directory and If this flag isdisabled post-install, the cluster may be DESTROYED.
+      # If the CRDs are deleted in this case, see the disaster recovery guide to restore them.
+      # https://rook.io/docs/rook/latest/Troubleshooting/disaster-recovery/#restoring-crds-after-deletion
+      enabled: true
+
+    # Whether rook watches its current namespace for CRDs or the entire cluster, defaults to false
+    currentNamespaceOnly: true
```

Comment:
why not watch all namespaces by default?

Reply:
we can test deploying another cephcluster in another namespace or even same namespace and once everything works, we can flip this boolean.

Reply:
sounds good to me